pkgs/games/freedroid/default.nix

```
@@ -0,0 +1,57 @@
+{ lib, stdenv
+, fetchFromGitHub
+, makeDesktopItem, copyDesktopItems
+, imagemagick
+, autoconf, automake
+, SDL, SDL_mixer, SDL_image, SDL_gfx
+, libvorbis
+, libjpeg, libpng
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "freedroid";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner  = "ReinhardPrix";
+    repo   = "FreedroidClassic";
+    rev    = "release-${version}";
+    sha256 = "027wns25nyyc8afyhyp5a8wn13x9nlzmnqzqyyma1055xjy5imis";
+  };
+
+  nativeBuildInputs = [ autoconf automake copyDesktopItems imagemagick ];
+  buildInputs = [
+    SDL SDL_image SDL_gfx
+    (SDL_mixer.override { enableMusicMod = true; })
+    libjpeg libpng libvorbis zlib
+  ];
+
+  postPatch = "touch NEWS";
+
+  preConfigure = "./autogen.sh";
```

```suggestion
  preConfigure = ''
    ./autogen.sh
  '';
```

Can we use autoReconfHook here?

---

Yes, we can :-) Thanks!